src/test/java/io/pravega/connectors/flink/PravegaConfigTest.java

```diff
@@ -78,6 +78,19 @@ public void testScopePriority() {
         assertEquals("scope2", config.getDefaultScope());
     }
 
+    @Test
+    public void testScopeSet() {
```

Comment:
We can add this case into the above one. This is also part of the priority check.